riotfile.py

```diff
@@ -103,7 +103,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         "DD_REMOTE_CONFIGURATION_ENABLED": "false",
         "DD_CIVISIBILITY_AGENTLESS_ENABLED": "1",
         "DD_CIVISIBILITY_CODE_COVERAGE_ENABLED": "1",
-        "DD_CIVISIBILITY_ITR_ENABLED": "1",
+        "DD_CIVISIBILITY_ITR_ENABLED": "0",
```

Comment:
Yes, this will be removed before merge. It disabled Intelligent Test Runner, which wasn't running all tests I needed

Reply:
@wconti27 , can you list some of the tests that were being skipped, by any chance?

Reply:
@romainkomorndatadog This [pipeline](https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/57218) should've been running tests for Kafka, Kombu, and Botocore. I suspect since all changes were contained to the `ddtrace/internal/datastreams` respective files for those integrations, that they were being skipped since changes weren't directly in the contrib.

Reply:
@wconti27 , if you're talking about the Kafka, Kombu, and Botocore suites that CircleCI runs, the decision on whether or not they run is made by the dependencies defined in https://github.com/DataDog/dd-trace-py/blob/main/tests/.suitespec.json , and changing the `DD_CIVISIBILITY_ITR_ENABLED` env var only affects that by coincidence because you're touching `riotfile.py`.

If those contribs depend on `datastreams`, that needs to be reflected in their component in the suite spec, eg: [botocore](https://github.com/DataDog/dd-trace-py/blob/main/tests/.suitespec.json#L669-L680)'s would have to be updated to something like
```
        "botocore": [
            "@bootstrap",
            "@core",
            "@contrib",
            "@tracing",
            "@llmobs",
            "@botocore",
            "@llmobs",
            "@datastreams",
            "tests/contrib/botocore/*",
            "tests/contrib/boto/*",
            "tests/snapshots/tests.contrib.botocore*"
        ],
```

Reply:
cool, I'll make those changes!